.github/workflows/ci-and-e2e-tests.yaml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - '**'
 
+env:
+  CLOUDSERVER_IMAGE: ${{ vars.CLOUDSERVER_IMAGE }}
```

Comment:
Well that's an interesting move. So you want to be able to modify cloudserver's image without making a PR?
I don't know to which frequency you will want to update that image but I feel like this will make it harder to debug or modify it.

If the problem you are aiming to solve, is to frequently make updates to cloudserver I would suggest the following: (And I can give you a hand to set this up)

- Hardcode the full image url in the `docker-compose.yaml` file.
- Configure dependabot to look for this file and make frequent update to it.
- Add a similar workflow just like the one we added on cosi.

Reply:
Reference ticket: https://scality.atlassian.net/browse/S3CSI-12